dynamiq/nodes/tools/firecrawl.py

```diff
@@ -30,17 +30,21 @@ class ExtractorOptions(BaseModel):
     extraction_schema: dict | None = None
 
 
+class FirecrawlInputSchema(BaseModel):
+    url: str = Field(..., description="Parameter to specify the url of the page to be scraped.")
```

Comment:
This field is required, so `FirecrawlTool.url` became unusable. Please make it oprional

Reply:
It functions as previously. In case url will be empty we will stick to FirecrawlTool.url

Reply:
Yes, If it is missed, but not when it present but empty.

Reply:
I made this parameter in schema optional.